### ISSUE TITLE
About the pre-training checkpoint you released

### ISSUE THREAD

**Issue by kerryhhh** [proteus1991/PSCC-Net#12]
Hello! I would like to ask on which dataset the pre-trained model released in the repository is trained? Also can you release the test code used to calculate the evaluation metrics?

**Comment by ORainn** [comment]
Same question. I also want to know the dataset used to train this pre-trained model.

**Comment by xiong76** [comment]
> Same question. I also want to know the dataset used to train this pre-trained model.

How do you have the floder of mask of splice?I failed in training because there is no floder of mask,Is this datasets incomplete?

**Comment by ORainn** [comment]
> > Same question. I also want to know the dataset used to train this pre-trained model.
> 
> How do you have the floder of mask of splice?I failed in training because there is no floder of mask,Is this datasets incomplete?

The dataset is complete. I followed the tips in this repo.

**Comment by proteus1991** [comment]
> Hello! I would like to ask on which dataset the pre-trained model released in the repository is trained? Also can you release the test code used to calculate the evaluation metrics?

Hi, you can find the training dataset in [Baidu Cloud](https://pan.baidu.com/s/1jbpPNp7UtnKo9OnZeEm1Yw), which was used to train the pre-trained model. More details can be found in our paper. 

I used the [scikit-learn](https://scikit-learn.org/) package to evaluate the results, which is simple and straightforward.
